src/content/test/errors/status-messages.tsx

```diff
@@ -0,0 +1,162 @@
+            change of focus. People who have good vision can be expected to notice when a status message is added to the viewport, but
+            people who are blind or have low vision can't. Instead, assistive technologies must be able to programmatically determine when a
+            status message is provided.
+        </p>
+
+        <h2>How to fix</h2>
+        <p>Ensure any status message that's displayed as visible text has the appropriate ARIA role: </p>
+        <ul>
+            <li>
+                If the status message contains important, time-sensitive information that should be communicated to users immediately
+                (potentially clearing the speech queue of previous updates), use
+                <Markup.HyperLink href="https://www.w3.org/TR/wai-aria-1.2/#alert">role="alert"</Markup.HyperLink>, which has an implicit
+                <Markup.HyperLink href="https://www.w3.org/TR/wai-aria-1.2/#aria-live">aria-live</Markup.HyperLink>
+                value of <Markup.Emphasis>assertive</Markup.Emphasis>.
+            </li>
+            <li>
+                Otherwise, use a role with an implicit aria-live value of <Markup.Emphasis>polite</Markup.Emphasis>:
+                <ul>
+                    <li>
+                        Use <Markup.HyperLink href="https://www.w3.org/TR/wai-aria-1.2/#status">role="status"</Markup.HyperLink> for a
+                        simple status message that's not urgent enough to justify interrupting the current task.
+                    </li>
+                    <li>
+                        Use <Markup.HyperLink href="https://www.w3.org/TR/wai-aria-1.2/#log">role="log"</Markup.HyperLink> if new
+                        information is added to the status message in meaningful order, and old information might disappear (such as a chat
+                        log, game log, error log, or messaging history).
+                    </li>
+                    <li>
+                        Use <Markup.HyperLink href="https://www.w3.org/TR/wai-aria-1.2/#progressbar">role="progressbar"</Markup.HyperLink>
+                        if the message conveys the status of a long-running process.
+                    </li>
+                </ul>
+            </li>
+        </ul>
+        <h2>Example</h2>
+        <Markup.PassFail
+            failText={
+                <p>
+                    A form requires the user to enter their username. If the user fails to enter a username, a script displays a status
+                    message as visible text. However, the text does not have an appropriate ARIA role, so assistive technologies do not
+                    announce it.
+                </p>
+            }
+            failExample={`
+                <body>
+                <form name="enroll" id="enroll" method="post" action="">
+                <p [id="error"]>You must enter your username.</p>
+                <p>
+                <label for="email">Username (required): </label>
```

Comment:
Should the for attribute on this label be `"username"` to match the input element? (same in passExample)

Reply:
I think you're correct...